tests/inlineasm/asmspecialregs.py

```diff
@@ -0,0 +1,24 @@
+import pyb
+import time
```

Comment:
We should keep inlineasm tests free of non-core imports, like pyb and time.  So I'd say remove the test requiring pyb.Timer.  It's likely to be flaky anyway (and it's the reason why Travis fails, due to qemu-arm build not having pyb module).